pm_muairss_write/pm_muairss_write.xml

```diff
@@ -23,19 +23,19 @@
             grep -o "name" params.yaml | wc -l > tree.txt &&
         #end if
         find \$out_folder >> tree.txt
-        
+
     ]]></command>
     <inputs>
         <param type="data" name="structure" format="cell" />
         <param type="hidden" name="testing" value="false"/>
-        <param type="data" name="params" format="txt" />
+        <param type="data" name="params" format="yaml" />
     <conditional name="iscastep">
         <param type="boolean" value="false" name="iscastep_sel" label="Is the calculator castep?">
             <option value="false">no</option>
             <option value="true">yes</option>
-        </param>   
+        </param>
         <when value="true">
-            <param type="data" name="castep_param" format="txt" />
+            <param type="data" name="castep_param" format="txt,yaml" />
```

Comment:
.param files seem to get parsed as YAML so this seems like the safest option

Reply:
ok